install.sh

```diff
@@ -78,20 +81,30 @@ test_dep "dot" 0 "graphviz"
 
 test_dep "npm" 0 "npm"
 
-test_dep "bower" 1
+test_dep "bower" 1 2>/dev/null
 if [ $RETVAL -eq 1 ]; then
      print_need_dep "bower"
      echo "sudo npm install -g bower"
      exit 1
+else
+     BOWER=$(bower -v)
+     if [ -z "$BOWER" ]; then
+        echo "If you are running distro debian based machine run:"
+        echo "sudo apt-get install nodejs-legacy"
+        exit 1
+    fi
 fi
 
-npm install --save
+npm install
 bower install
 echo "Installing required services..."
 SERVER_PATH=$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)
 su -c "cp $SERVER_PATH/scripts/units/fbp-runner@.service /lib/systemd/system/"
 su -c "cp $SERVER_PATH/scripts/units/soletta-dev-app-server.service.in /lib/systemd/system/soletta-dev-app-server.service"
 su -c "sed -i "s@PATH@"$SERVER_PATH"@" /lib/systemd/system/soletta-dev-app-server.service"
+if [ $NODE_TYPE -eq 1 ]; then
+    su -c "sed -i "s@"node"@"nodejs"@" /lib/systemd/system/soletta-dev-app-server.service"
```

Comment:
You could handle this in a different way:
Use a placeholder in soletta-dev-app-server.service (e.g. NODE_BIN_NAME). NODE_TYPE (which is a funny name, considering Soletta's node types ;) would contain the binary name itself. This would make the script easier to update, in case of another binary name for nodejs (e.g. njs).